buildkitd/buildkitd.go

```diff
@@ -11,13 +11,14 @@ import (
 	"sync"
 	"time"
 
+	"github.com/moby/buildkit/client"
+	_ "github.com/moby/buildkit/client/connhelper/dockercontainer" // Load "docker-container://" helper.
+	"github.com/pkg/errors"
+
```

Comment:
My editor keeps wanting to sort these imports, but it looks like we usually keep them organized like this anyway?

Reply:
I suspect it's just some old unorganised imports -- looks like a good change.